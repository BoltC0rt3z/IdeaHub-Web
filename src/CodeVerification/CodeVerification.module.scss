@import '../styles/styles.module.scss';

.container {
  background: $color-light-gray;
  min-height: 100vh;

  .codeVerification {
    padding-top: 32px;
    padding-bottom: 32px;

    .card {
      background: $color-light;
      border-radius: 8px;
      margin-left: 16px;
      margin-right: 16px;
      box-shadow: 2px 2px 2px 2px $color-light-shadow;

      .title {
        font-weight: bold;
        font-size: 20px;
        border-bottom: 2px solid $color-light-gray;
        padding: 16px;
        color: $color-dark-blue-lighter;
      }

      .text {
        padding: 16px 16px 0px 16px;
        color: $color-dark-blue-lighter;

        .mail {
          font-weight: bold;
        }
      }
    }

    .inputRow {
      padding: 0px 16px 16px 16px;
    }

    .resendMail {
      cursor: pointer;
      padding: 16px;
      color: dodgerblue;
      font-size: 16px;
    }

    .buttonsRow {
      padding: 16px;
      border-top: 2px solid $color-light-gray;

      .buttonColumn {
        padding: 8px;
      }
    }
  }
}

@mixin button {
  background: #ebedf0;
  border: none !important;
  font-size: 12px;

  &:hover,
  &:active,
  &:focus {
    background: #ebedf0;
  }
}

.button {
  @include button();
}

.disabledButton {
  @include button();
  color: #bec3c9 !important;
}
