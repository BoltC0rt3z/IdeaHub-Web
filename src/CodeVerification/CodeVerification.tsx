import React from 'react';

const CodeVerification: React.FC = () => {
  return <div>Verification code</div>;
};

export default CodeVerification;
