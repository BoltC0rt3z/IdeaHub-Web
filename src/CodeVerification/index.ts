import CodeVerification from './CodeVerification';

export { CodeVerification };
