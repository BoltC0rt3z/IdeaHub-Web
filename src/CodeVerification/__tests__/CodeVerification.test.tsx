import React from 'react';
import { mount, ReactWrapper } from 'enzyme';
import { Provider } from 'react-redux';
import createMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import CodeVerification from '../CodeVerification';

describe('Tests for code verification page', () => {
  const store = createMockStore([thunk])({});
  const wrapper: ReactWrapper = mount(
    <Provider store={store}>
      <CodeVerification />
    </Provider>,
  );

  it('should render component without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
