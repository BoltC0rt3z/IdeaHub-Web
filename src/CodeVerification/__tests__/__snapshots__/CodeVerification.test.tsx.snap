// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for code verification page should render component without crushing 1`] = `ReactWrapper {}`;
